content/how-to-use-brotli-compression-method/index.md

```diff
@@ -0,0 +1,175 @@
+### Introduction
+
+Data compression is the process of encoding files to reduce their size, making them take up less space on your server and allowing them to be transferred to site clients more quickly. Compression reduces the amount of bandwith required for website rendering, which speeds up the process. Static assets such as *.js*, *.css* and *.json* text files are commonly used in web application projects, with redundancy and compression playing an important role.

```

Comment:
Requested changes not addressed. Refer your previous PR

Add objectives of this tutorial.

Add table of contents